packages/nextjs/src/index.server.ts

```diff
@@ -47,11 +47,18 @@ function sdkAlreadyInitialized(): boolean {
   return !!hub.getClient();
 }
 
-const SOURCEMAP_FILENAME_REGEX = /^.*\/\.next\//;
+/**
+ * WARNING: don't refactor this variable.
+ * This variable gets overridden at build time to set the correct path -- what users
+ * defined in the `distDir` option in their Next.js configuration.
+ */
+export const PROJECT_BASEPATH = '.next';
```

Comment:
I wonder if this should just go in it's own file - and then we can even snapshot test (https://jestjs.io/docs/snapshot-testing) against potential regressions.

Reply:
IMO, yes. There's more code related to integrations that should be moved to another file, like the `RewriteFrames` integration itself and the `addServerIntegrations` below, and also the new code in the `webpack` file. However, that refactor isn't part of this PR and should go in another one.